test/TesterInternal/MembershipTests/MembershipTableTestsBase.cs

```diff
@@ -102,43 +103,56 @@ protected virtual string GetAdoInvariant()
             Assert.True(entries.Contains(membershipEntries[9].SiloAddress.ToGatewayUri().ToString()));
         }
 
-        protected async Task MembershipTable_ReadAll_EmptyTable()
+        protected async Task MembershipTable_ReadAll_EmptyTable(bool extendedProtocol = true)
         {
             var data = await membershipTable.ReadAll();
             Assert.NotNull(data);
 
-            logger.Info("Membership.ReadAll returned VableVersion={0} Data={1}", data.Version, data);
+            if (extendedProtocol)
```

Comment:
I understand that your did not implement the extendedProtocol . That is fine, but it does not mean you don't want those checks. You do want most of those. Like that particular one `Assert.NotNull(data.Version.VersionEtag);` - yes, you do want this check. There should still be a valid non null Version. I actually think that you will need most of the checks that you now disabled.